compiler/onecc-docker/onecc-docker.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+from distutils.version import StrictVersion
+import json
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+
+def main():
+    installed_onecc_version = sys.argv[1]
+    versions_str = subprocess.check_output("curl -s https://api.github.com/repos/Samsung/ONE/tags", shell=True ,encoding='utf-8')
```

Comment:
How about use python library? We don't have to run bash command here.
```suggestion
import requests

response = requests.get("https://api.github.com/repos/Samsung/ONE/tags")
```

Reply:
I'll use requests instead of subprocess! Thank you

Reply:
Please consider the case when the response has a problem. I'm not sure but you can refer to `response.status`, `response.reason` information.